lnrpc/routerrpc/config_active.go

```diff
@@ -45,3 +63,24 @@ type Config struct {
 	// main rpc server.
 	RouterBackend *RouterBackend
 }
+
+// DefaultConfig defines the config defaults.
+var DefaultConfig = Config{
```

Comment:
would recommend defining this a function that returns the default config, o/w it can be modified by importers arbitrarily

Reply:
I am not too worried about that, but it is an improvement. Fixed.